src/lib/Scenes/Artwork/Components/__tests__/ArtworkDetails-tests.tsx

```diff
@@ -0,0 +1,31 @@
+import { mount } from "enzyme"
+import React from "react"
+import { ArtworkDetails } from "../ArtworkDetails"
+
+jest.unmock("react-relay")
+
+describe("Artwork Details", () => {
+  it("renders fields correctly", () => {
```

Comment:
May I ask for a couple more tests to verify the behavior of:
- If there are 3 or fewer fields available, it doesn't show the "Show more" link
- If there are > 3, it'll show that link and clicking it expands to show everything